ideas/310-core-team-proposal.md

```diff
@@ -72,72 +71,84 @@ See https://github.com/status-im/status-react/issues/6757 as an example of a sim
-- post-meeting report in a Discuss thread
+Goals:
+- Figure out the list of tasks and top UX issues that needs to be fixed
+- Rank them by priority (what should we do first)
 
-Post-project retrospective with post-mortem
-- when the project milestone is done/failed
-- when we are deciding to switch to another focus area
+## Specification
 
-[DRIs](https://www.forbes.com/sites/quora/2012/10/02/how-well-does-apples-directly-responsible-individual-dri-model-work-in-practice/#38840682194c) for focus areas / maintenance tasks:
-- single point of contact
-- focused on one thing (one can be a DRI only to one focus area/maintenance task)
+### 2. "Save Password" on Android
 
-# Core: Maintenance
+* Timeline - January, 14th
 
-#### 1. Battery usage
+The goal is to achieve feature-parity between iOS and Android in terms of
+storing the password.
 
-#### 2. Application performance profiling
+Epic: https://github.com/status-im/status-react/issues/5793
 
-#### 3. Mobile Releases
+- [ ] Threat modelling (STRIDE) + document about that.
 
-- [Mobile release guide](https://notes.status.im/mobile-release-guide)
-- [Mobile release notes](https://notes.status.im/mobile-release-notes)
-- Upgrade policy
-  - *User’s data.* User should be able to successfully upgrade preserving all his/her data from 0.9.19 to any beta release.
-  - *Messaging/Transactions.* User should be able to successfully interact (make transactions, chat, etc) with users that have 0.9.20 or later release.
+## Implementation
 
-#### 4. Product Analytics
+### 0. Meta
+* Timeline - EOQ2 2019, continuous process. Each backlog item discovered by
+    research will have it's own deadline.
 
-Product analytics provides insight into how Status is being used.  Currently we have 3 key metrics delivering insight:
+Setup a regular meetings with the stakeholders to adjust the backlog.
 
-- App downloads
-- Public message
-- ENS registrations
+`document progress` - keep the backlog and the state up to date with this
+document.
 
-The core team is responsible for maintaining the reporting of these metrics at [status.im/analytics](https://status.im/analytics/)
+Projects from should be put in the correct state to see the progress, etc.
 
-# Core: Projects - Active
+### 1. Message Reliability Improvements
 
-#### Message Reliability Improvements
+* Timeline - January, 11
 
 The goal of this project is:
 * to make messaging extremely reliable;
 * to be able to prove that;
 * to be able to show that information to a user (especially it is important to
     communicate if the message isn't delivered).
 
-See more and up-to-date info in the Epic: https://github.com/status-im/status-react/issues/6757
+`docyment progress`: see the epic https://github.com/status-im/status-react/issues/6757
+
+### 2. "Save Password" on Android
+
+* Timeline - January, 21th
+
+Epic: https://github.com/status-im/status-react/issues/5793
+
+- [ ] Implement according to the spec
+- [ ] Audit the implementation
+
+## Maintenance
```

Comment:
Is the idea that these tsections wil lbe specced out?

Reply:
As in: figuring out what is necessary for critical path in each department. E.g. BW we don't want to be release a public app only to have it kill peoples data plan. So figuring out the root cause of this would be under research

Reply:
ah, there are projects in our backlog that would be marked as critical for the release